### DISCUSSION TITLE
Error in comfyui, please help

### DISCUSSION THREAD

**Discussion by mmak0** [comfyanonymous/ComfyUI#4587] [Q&A]
Hi, since yesterday I have this error in my comfyui, how to fix this ?

C:\Users\max\AI GENERATE\ComfyUI_windows_portable_nvidia\ComfyUI_windows_portable\ComfyUI\comfy\ldm\modules\attention.py:407: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.) out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False) Requested to load AutoencodingEngine Loading 1 new model loaded completely 0.0 159.87335777282715 True 
Warning torch.load doesn't support weights_only on this pytorch version, loading unsafely. 
Requested to load Flux

PyTorch version: 2.3.1+cu121
CUDA version (PyTorch built with): 12.1
CUDA runtime version: 8.6
GPU: RTX 3060 12 GB
Model used: Flux NF4 v2

When I upgrade pytorch to 2.4, comfyui doesn't work

**Comment by ltdrdata** [comment]
1. Is it correct to load the NF4 model using `CheckpointLoaderNF4`?
2. Does this also occur when using models that are not NF4?

**Reply by mmak0** [reply]
1. Yes, NF4 is loaded with CheckpointLoaderNF4. But I need to switch to dev channel and fix bitsandbytes every time I run ComfyUI, otherwise I get error for NF4 loader.
2. This is happening on GGUF and NF4 models. Default Dev and Schnell are not working (but used to work before). I just get "Reconnecting" in top right corner without any logs in cmd. Last messages in cmd : 
got prompt
Using split attention in VAE
Using split attention in VAE
After this I'm unable to restart ComfyUI from browser, I need to close cmd and run it again
